mittab/libs/outround_tab_logic/checks.py

```diff
@@ -0,0 +1,114 @@
+from mittab.apps.tab.models import *
+from mittab.libs.errors import PrevRoundNotEnteredError
+
+
+def lost_teams():
+    losers = [outround.loser.id
```

Comment:
nit: why bother assigning this to a variable, can't we just return it inline?

Reply:
Done